src/vs/workbench/browser/dnd.ts

```diff
@@ -46,6 +48,11 @@ export class DraggedEditorGroupIdentifier {
 	constructor(readonly identifier: GroupIdentifier) { }
 }
 
+export class DraggedExtensionTreeItemsIdentifier {
```

Comment:
Should we make this more generic? Why so specific to extension trees? Maybe we want to use it for our trees as well, I suggest `DraggedTreeItemsIdentifier`.